src/ReactiveUI.Uno/ReactiveUI.Uno.csproj

```diff
@@ -0,0 +1,33 @@
+﻿<Project Sdk="MSBuild.Sdk.Extras">
+  <PropertyGroup>
+    <TargetFrameworks>xamarinmac20;MonoAndroid90;xamarinios10;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition=" '$(OS)' == 'Windows_NT' ">$(TargetFrameworks);uap10.0.17763</TargetFrameworks>
+    <PackageId>ReactiveUI.Uno</PackageId>
+    <Title></Title>
+    <Description>Uno-Platform specific extensions to ReactiveUI</Description>
```

Comment:
Is it _for ReactiveUI_ ? My english may be at fault :)

Reply:
It should be _for_ ReactiveUI

Reply:
Also, we're using "Uno Platform" without the dash.

Reply:
I see you resolved this @ghuntley, thank you so much!
was actually doubting if i should cc u cuz i didn't wanna bother
you guys rock 👑